### DISCUSSION TITLE
Issue with .host and .digital domain?

### DISCUSSION THREAD

**Discussion by mbparvezme** [vercel/community#4071] [Help]
### Summary

I have two projects where I use the following top-level domains (TLDs):
- .digital
- .host

In both cases, my .com domains are functioning correctly with the same configuration and the same projects. However, the .digital and .host TLDs are not working at all. On Vercel, both domains are displaying the following message:

> Valid Configuration / Assigned to master / Good news! Your DNS records are set up correctly, but it can take some time for them to propagate globally.

I have tried several times and checked both of the domains on https://intodns.com/ (showing no errors), but got the same results. My question is, is there any issue with the above TLDs? What could be the issue?

### Steps to Reproduce

A record
Name: @
Value: 76.76.21.21

CNAME record
Name: www.MY_DOMAIN.host / www.MY_DOMAIN.digital
Value: cname.vercel-dns.com

NS record
Registry provided default record

**Comment by amyegan** [comment]
How recently did you add the domains to your project?

**Comment by mbparvezme** [comment]
Actually, I've tried a month ago. Everything seems okay but just the domain
is not working.

Please see the image given below. If you need more information, please let
me know.
[image: Screenshot 2023-10-15 131010.png]

**Reply by amyegan** [reply]
Hey @mbparvezme. That image didn't come through for me. Would you mind sharing it directly in our GitHub discussion? If you let me know the domains, I can also check them for errors.

**Comment by mbparvezme** [comment]
Well! My domain is "bipby.digital". Does it help you?

**Reply by amyegan** [reply]
Thanks for sharing that. It looks like bipby.digital uses Cloudflare https://www.whatsmydns.net/#NS/bipby.digital

This guide has some setup and troubleshooting steps for Cloudflare domains: https://vercel.com/docs/integrations/cloudflare
